Algorithms.Tests/Numeric/KrishnamurthyNumberCheckerTests.cs

```diff
@@ -0,0 +1,40 @@
+using Algorithms.Numeric;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Numeric
+{
+    [TestFixture]
+    public class KrishnamurthyNumberCheckerTests
+    {
+        [Test]
```

Comment:
Test attribute is redundant when TestCase attribute is already specified

Reply:
Fixed in the new commit

Reply:
It's still here 
![image](https://user-images.githubusercontent.com/29201949/193800708-705dd3cb-d178-4ad6-8720-9a570502dbc0.png)